addons/bestja_detailed_reports/models.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api, exceptions
+from dateutil import parser
+
+
+class Project(models.Model):
+    _inherit = 'bestja.project'
+
+    detailed_reports = fields.One2many('bestja.detailed_report', inverse_name='project')
+    use_detailed_reports = fields.Boolean(required=True, string="Raporty szacunkowe do zbiórki żywności", default=True)
```

Comment:
Also, I don't think it should be checked by default. It should be unchecked by default, but only visible in top most project.